action/tx_legacy.go

```diff
@@ -18,6 +18,7 @@ var _ TxCommonInternal = (*LegacyTx)(nil)
 
 // LegacyTx is the legacy transaction
 type LegacyTx struct {
+	version  uint32
```

Comment:
we can define a new Web3LegacyTx to handle this

Reply:
updated